keyboards/keebio/levinson/rev2/config.h

```diff
@@ -51,8 +51,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 /* ws2812 RGB LED */
 #define RGB_DI_PIN D3
-
 #define RGBLED_NUM 12    // Number of LEDs
+#define RGBLED_SPLIT { 6, 6 }
```

Comment:
From <https://docs.keeb.io/adding-rgb-underglow/> where the ws2812 signal is passed over trrs, won't this break existing users?

Reply:
Yeah, it may